.github/workflows/test-windows.yml

```diff
@@ -79,6 +79,7 @@ jobs:
         pytest
         pytest-cov
         pytest-timeout
+        trove-classifiers>=2024.10.12
```

Comment:
These things are essentially the test dependencies, `trove-classifiers>=2024.10.12` being required for a `pyroma` test.

Can we list this new `trove-classifiers>=2024.10.12` in `pyproject.toml`'s `optional-dependencies.tests`, remove this step (and duplicated dependencies), and modify the `Build Pillow` step below to `pip install` with the `tests` extra?

```diff
-        cmd /c "winbuild\build\build_env.cmd && $env:pythonLocation\python.exe -m pip install -v $FLAGS ."
+        cmd /c "winbuild\build\build_env.cmd && $env:pythonLocation\python.exe -m pip install -v $FLAGS .[tests]"
```